src/tools.ts

```diff
@@ -269,3 +269,58 @@ export async function collapseCodeSections (editor: monaco.editor.IStandaloneCod
     }
   }
 }
+
+export function mapClipboard (
+  editor: monaco.editor.IStandaloneCodeEditor, {
+    toClipboard = v => v,
+    fromClipboard = v => v
+  }: {
+    toClipboard: (v: string) => string
+    fromClipboard: (v: string) => string
+  }): void {
+  let copiedText = ''
```

Comment:
It's not super clear why we have to maintain a `copiedText`.

Reply:
I simplified the function, it still works on chrome and firefox but I suspect this code was to make it work on edge

Reply:
Even now that Edge is Chromium based ?

Reply:
Probably not... I don't know and I don't have the possibility to check, nobody has windows there